src/components/maincontroller.ts

```diff
@@ -154,14 +150,20 @@ window.addEventListener('beforeunload', () => {
     reportPlaybackStopped($scope, getReportingParams($scope));
 });
 
-mgr.addEventListener(cast.framework.events.EventType.PLAY, (): void => {
-    play($scope);
-    reportPlaybackProgress($scope, getReportingParams($scope));
-});
+window.playerManager.addEventListener(
+    cast.framework.events.EventType.PLAY,
+    (): void => {
+        play($scope);
+        reportPlaybackProgress($scope, getReportingParams($scope));
+    }
+);
 
-mgr.addEventListener(cast.framework.events.EventType.PAUSE, (): void => {
-    reportPlaybackProgress($scope, getReportingParams($scope));
-});
+window.playerManager.addEventListener(
+    cast.framework.events.EventType.PAUSE,
+    (): void => {
+        reportPlaybackProgress($scope, getReportingParams($scope));
+    }
+);
```

Comment:
I'm guessing this was done to reduce the line length but the previous style was much more readable in my opinion.

Reply:
It was previously `mgr.addEventListener` which was refactored to `window.playerManger`. prettier is unfortunately the master of the line breaks here :|